.github/workflows/pull-request.yaml

```diff
@@ -1,64 +1,66 @@
 name: Unikorn Push
 on:
   pull_request:
+    paths-ignore:
+      - "README.md"
```

Comment:
Rather than hard coding a single path, wouldn't it be a lot better to just say *.md?

Reply:
Yea let's go for that